### PR TITLE
Technical review of PR #804

### LINKED ISSUE #805: Tech review for merged PR https://github.com/open-life-science/open-life-science.github.io/pull/804

### PR DESCRIPTION
**Pull request by bebatut**
Fixes #805

### CODE REVIEW

File: _data/people.yaml [hunk 1]
```diff
@@ -8371,11 +8371,10 @@ rossmounce:
   website: https://rossmounce.co.uk/
   country_3: GBR
   continent: Europe
-rowland-mosbergen:
```
Comment (NPDebs) [inline _data/people.yaml:8374]: Definitely the more sensible way to do it! 😆

File: _posts/2024-07-12-RSE-AUNZ-joins-OLS.md [hunk 2]
```diff
@@ -31,12 +32,12 @@ If you want to join the community, sign-up to the [Google Groups mailing list](h
...
 
 ## Fiscal Sponsorship with OLS
 
-Fiscal sponsorship is a relatively new program for OLS, mainly in response to the fact that people participating in the OLS mentorship programs ([Open Seeds](https://we-are-ols.org/openseeds/) and [Nebula](https://we-are-ols.org/nebula/)) often desire to take "next steps" for their communities or projects, but may not have the legal infrastructure to do so (e.g. ways to handle money, accounting, a registered organisation, etc.), or may have an institution they can work with that may not have the desired flexibility to meet their needs. 
+Fiscal sponsorship is a relatively new program for OLS. It was started mainly in response to the fact that people participating in the OLS mentorship programs ([Open Seeds]({% link openseeds/index.md %}) and [Nebula]({% link nebula/index.md %})) often desire to take "next steps" for their communities or projects, but may not have the legal infrastructure to do so (e.g. ways to handle money, accounting, a registered organisation, etc.), or may have an institution they can work with that may not have the desired flexibility to meet their needs.
```
Comment (NPDebs) [inline _posts/2024-07-12-RSE-AUNZ-joins-OLS.md:41]: I have an idea - we get a large cookie jar and put a dollar every time we don't use the `link` tag. :)

**Review by NPDebs — APPROVED** [review]
@bebatut I don't know what we would do without your keen eyes! Thanks so much for these fixes. 🙌

File: _posts/2024-07-12-RSE-AUNZ-joins-OLS.md (hunk 2 above)
Comment (bebatut) [reply _posts/2024-07-12-RSE-AUNZ-joins-OLS.md:41]: We will get a lot of cookies at the end of the year 😋

**Pull request merged**